cmd/gcp-controller-manager/node_syncer.go

```diff
@@ -252,3 +259,34 @@ func (nm *nodeMap) gsaEmailsByNode(node string) ([]gsaEmail, error) {
 	}
 	return l, nil
 }
+
+type nodeZones struct {
+	sync.RWMutex
+	m      map[string]string
+	client clientset.Interface
+}
+
+func newNodeZones(client clientset.Interface) *nodeZones {
+	return &nodeZones{
+		m:      make(map[string]string),
+		client: client,
+	}
+}
+
+func (nz *nodeZones) zoneByNode(name string) (string, error) {
```

Comment:
what is name here? Is it the node name?

Reply:
It's nodeName. I read it as zoneByNodeName. Since it causes confusion, let's use nodeName.

> @wyuan1704 - do we need to add new test cases for the new behavior?

It doesn't change the public behavior, so I didn't add a new test case. But I modified the test with fake.NewSimpleClientset(node), so that it hits this function.